Abusing CSharp/Code/LinqToOperators/OperatorEnumerable.cs

```diff
@@ -286,34 +286,16 @@ public override int GetHashCode()
 
         private static IEnumerable<object> RepeatSequence(IEnumerable<object> source, int times)
         {
-            for (int i = 0; i < times; i++)
-            {
-                foreach (var item in source)
-                {
-                    yield return item;
-                }
-            }
+            return Enumerable.Range(1,times)
```

Comment:
Not quite sure why I didn't do this before. It might be better as

```csharp
Enumerable.Repeat(source, times).SelectMany(x => x);
```

No variable capture that way.

Reply:
Ahh, good call. I didn't recall that Enumerable.Repeat existed. Thanks for pointing that out.

Reply:
> Enumerable.Repeat(source, times).SelectMany(x => x);

Well, `.SelectMany(x => x)` won't work on a *object* cc @jskeet

Reply:
But the type of `x` would be `IEnumerable<object>`, so it's fine.